README.md

```diff
@@ -1,24 +1,20 @@
-[![Build Status](https://semaphoreci.com/api/v1/calico/libcalico-go/branches/master/badge.svg)](https://semaphoreci.com/calico/libcalico-go)[![Slack Status](https://slack.projectcalico.org/badge.svg)](https://slack.projectcalico.org) [![IRC Channel](https://img.shields.io/badge/irc-%23calico-blue.svg)](https://kiwiirc.com/client/irc.freenode.net/#calico)
+[![Build Status](https://semaphoreci.com/api/v1/calico/libcalico-go/branches/master/shields_badge.svg)](https://semaphoreci.com/calico/libcalico-go) [![Slack Status](https://slack.projectcalico.org/badge.svg)](https://slack.projectcalico.org) [![IRC Channel](https://img.shields.io/badge/irc-%23calico-blue.svg)](https://kiwiirc.com/client/irc.freenode.net/#calico) [![GoDoc](https://godoc.org/github.com/golang/gddo?status.svg)](http://godoc.org/github.com/projectcalico/libcalico-go)
 
 # libcalico-go
 This repositiory contains Calico's Go components:
 
 - `libcalico`, which can be imported as `"github.com/projectcalico/libcalico-go/lib"`
 
-This library is used by both `calicoctl`, `calico-cni` and `felix`.
-
-If you are looking for the golang version of `calicoctl` it is in the process of moving and
-can be found in our
-[calico-containers repo / golang branch](https://github.com/projectcalico/calico-containers/tree/golang).
+This library is used by [calicoctl](https://github.com/projectcalico/calicoctl), [Calico cni-plugin](https://github.com/projectcalico/calico-cni), [Calico libnetwork-plugin](https://github.com/projectcalico/libnetwork-plugin) and [felix](https://github.com/projectcalico/felix).
```

Comment:
cni-plugin -> CNI plugin
libnetwork-plugin -> libnetwork plugin   (don't think we need the hyphen do we?)